### PR TITLE
[MXNet][Test]Updating the requirement and location of the test

### PR DESCRIPTION
**Pull request by TusharKanekiDey**
*Issue #, if available:*

## Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

*Description:*
It fixes the bug (https://github.com/aws/deep-learning-containers/issues/268)
*Tests run:*

*DLC image/dockerfile:*

*Additional context:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/sagemaker_tests/mxnet/inference/integration/local/local_mode_utils.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 import tarfile
 import time
 
-from test.integration import RESOURCE_PATH
+from ...integration import RESOURCE_PATH
```
Comment (YYStreet) [inline test/sagemaker_tests/mxnet/inference/integration/local/local_mode_utils.py:21]: Is it a good practice to use relative path for importing? Please also talk with other experienced engineers for it.

Comment (TusharKanekiDey) [reply test/sagemaker_tests/mxnet/inference/integration/local/local_mode_utils.py:21]: I have used #210 as base. I can use direct path if it is more advisable.

**Comment by TusharKanekiDey** [comment]
@ChaiBapchya I have added the description. About the test, we can have an offline discussion on it.

File: test/sagemaker_tests/mxnet/inference/integration/local/local_mode_utils.py (hunk 1 above)
Comment (Satish615) [reply test/sagemaker_tests/mxnet/inference/integration/local/local_mode_utils.py:21]: using absolute import was creating was creating a a longer path
from test/sagemaker_tests/trensorflow/tensorflow_training/integration import RESOURCE_PATH 

we can change to this in future if needed

File: test/sagemaker_tests/mxnet/training/integration/local/test_horovod.py [hunk 2]
```diff
@@ -18,8 +18,7 @@
 
 import pytest
 from sagemaker.mxnet import MXNet
-from test.integration import RESOURCE_PATH
-
+from ...integration import RESOURCE_PATH
```
Comment (Satish615) [inline test/sagemaker_tests/mxnet/training/integration/local/test_horovod.py:21]: test_horovod is recently added and it expects additional arguments while running the pytest commond. This wont be fixed by import change in the pr

**Pull request closed without merging**